packages/@glimmer/destroyable/index.ts

```diff
@@ -115,8 +115,9 @@ export function registerDestructor<T extends Destroyable>(
 
   let meta = getDestroyableMeta(destroyable);
 
-  let destructorsKey: 'eagerDestructors' | 'destructors' =
-    eager === true ? 'eagerDestructors' : 'destructors';
+  let destructorsKey: 'eagerDestructors' | 'destructors' = eager
```

Comment:
`x === true` boolean checks are autofixed to `x` boolean checks if `x` is a boolean.